content/flux/v0.x/data-types/composite/array.md

```diff
@@ -3,18 +3,118 @@ title: Work with arrays
 seotitle: Work with arrays in Flux
 list_title: Array
 description: >
-  ...
+  Learn how to work with arrays in Flux.
 menu:
   flux_0_x:
     name: Array
     parent: Composite types
 weight: 202
 flux/v0.x/tags: ["composite types", "data types"]
+related:
+  - /flux/v0.x/stdlib/universe/length/
+  - /flux/v0.x/stdlib/array/
 ---
 
-Notation
-  - reference items in an array with bracket notation
+An **array** type is an ordered sequence of values of the same type.
 
-Note - currently no way to iterate through an array
+## Array syntax
+An array literal contains a sequence of values (also known as elements) enclosed
+in square brackets (`[]`).
+Values are comma-separated and must be the same type.
 
-- Check if a value exists in an array (contains function)
\ No newline at end of file
+##### Example arrays
+```js
+["1st", "2nd", "3rd"]
+[1.23, 4.56, 7.89]
+[10, 25, -15]
+```
+
+## Reference values in an array
+Use **bracket notation** to reference reference a value in an array. 
+Flux arrays use **zero-based indexing**.
+Provide the index of the value to reference.
+
+```js
+arr = ["1st", "2nd", "3rd"]
+
+arr[0]
+// Returns 1st
+
+arr[2]
+// Returns 3rd
+```
+
+## Operate on arrays
+
+- [Iterate over an array](#iterate-over-an-array)
+- [Check if a value exists in an array](#check-if-a-value-exists-in-an-array)
+- [Get the length of an array](#get-the-length-of-an-array)
+- [Create a stream of tables from an array](#create-a-stream-of-tables-from-an-array)
+- [Compare arrays](#compare-arrays)
+
+### Iterate over an array
+{{% note %}}
+Flux currently does not provide a way to iterate over an array.
+{{% /note %}}
```

Comment:
Any alternative/workaround example to extracting this data some way programmatically?

Reply:
No, this is a known limitation of Flux right now. Honestly, the only reason I included this was because it gets asked about so often. This gives users something to land on, even if it doesn't solve their problem.